oak/server/oak_node.h

```diff
@@ -54,13 +60,14 @@ class OakNode final : public ::oak::Node::Service {
   // Incoming gRPC data for the current invocation.
   const ::grpc::GenericServerContext* server_context_;
 
-  std::unique_ptr<std::vector<char>> request_data_;
-  // Cursor keeping track of how many bytes of request_data_ have been consumed by the Oak Module
-  // during the current invocation.
-  uint32_t request_data_cursor_;
+  // Guards data used by OakRead and OakWrite host methods.
+  ::absl::Mutex module_data_mutex_;
+
+  // Span containing the gRPC request data read by the module.
```

Comment:
Could you clarify that this gets shifted as parts of the request are read?

Reply:
Done.